pandas/_testing/asserters.py

```diff
@@ -1382,3 +1382,20 @@ def assert_metadata_equivalent(
             assert val is None
         else:
             assert val == getattr(right, attr, None)
+
+
+def assert_mixed_int_string_entry(arr) -> bool:
```

Comment:
maybe "is_mixed_..." instead of "assert_mixed_..."?

Reply:
Thanks, I replaced `assert_mixed_int_string_entry` with `is_mixed_int_string_entry`.